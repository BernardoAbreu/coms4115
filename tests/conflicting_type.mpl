a = "134";
a = 3;