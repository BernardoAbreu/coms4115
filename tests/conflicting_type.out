Fatal error: exception Failure("cant change global a type")
